### DISCUSSION TITLE
Test containers MsSql module Database project script execute

### DISCUSSION THREAD

**Discussion by shaliake** [testcontainers/testcontainers-dotnet#999] [Q&A]
Hello! I would love to use test containers, but to utilise it I need to execute database project against mssql. What are my options? I have never used this nuget, so right now looking into options

**Comment by HofmeisterAn** [comment]
To create the database schema or execute the database migration, there are several options available. You can choose to use the Entity Framework, a database migration tool like Flyway, a custom Docker image, or alternatively, utilize the `WithResourceMapping(string, string)` API provided by Testcontainers, or one of its corresponding methods to copy scripts into the container prior to start (for this option the [following](https://github.com/microsoft/mssql-docker/tree/master/linux/preview/examples/mssql-customize) may also help).
